gapipy/models/base.py

```diff
@@ -126,7 +126,7 @@ def _set_resource_field(self, field, value):
             for v in value:
                 stub.append(self._model_cls(field)(v, stub=True))
         elif value:
-            stub = self._model_cls(field)(value, stub=True)
+            stub = self._model_cls(field)(value, self._client, stub=True)
         setattr(self, field, stub)
 
     def _set_resource_collection_field(self, field, value):
```

Comment:
Hm, ok, I'll look further into what that would require. I don't think this PR will be merged, but good for the conversation.